### DISCUSSION TITLE
Flashing GravityMon

### DISCUSSION THREAD

**Discussion by StodgyWaif** [mp-se/gravitymon#48] [General]
I don't know if it's recommended to do so but I flashed GravityMon on my iSpindels using the original iSpindel firmware config (Maintenance) page.
I had already calibrated them and didn't want to remove them from the containers to access the D1 Mini USB port.

Seems to be working well on all of them so far.

I didn't see this listed as an option on the main page so I thought I would mention it.

**Comment by mp-se** [comment]
That should work fine. The flash layout is the same and gravitymon is littlefs instead of spiffs (ispindle) but it will be formatted at startup. 

Thanks for the suggestion, i will add this option to the docs

**Comment by mp-se** [comment]
Inspired by your suggestion I have added an experimental feature for manual flashing. If you enter the url http://<device>/firmware.htm in the v0.9 release you can upload any firmware that uses the standard flash layout.
